binstar_client/requests_ext.py

```diff
@@ -8,11 +8,17 @@
 
 import requests
 import six
-from urllib3.filepost import choose_boundary, iter_fields
+from urllib3.filepost import choose_boundary
 
 logger = logging.getLogger('binstar.requests_ext')
 
 
+def iter_fields(fields):
+    if isinstance(fields, dict):
+        return ((key, val) for key, val in fields.items())
```

Comment:
Not that we really need to change what was in the source, but w/ python3, doesn't `fields.items()` already return an iterable of tuples?